[
  {
    "input": {
      "name": "Product1",
      "info": "Desc1",
      "price": "100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
    },
    "status_code": "exception",
    "response": "HTTPConnectionPool(host='127.0.0.1', port=8000): Max retries exceeded with url: /datatb/product/add/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fabe00560a0>: Failed to establish a new connection: [Errno 61] Connection refused'))",
    "mutation_type": "extremely_long_value",
    "request_id": 4,
    "timestamp": "2025-04-24T18:38:40.699056",
    "error": "HTTPConnectionPool(host='127.0.0.1', port=8000): Max retries exceeded with url: /datatb/product/add/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fabe00560a0>: Failed to establish a new connection: [Errno 61] Connection refused'))"
  },
  {
    "input": {
      "name": "Product1",
      "info": "Desc1",
      "price": "100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
    },
    "status_code": "exception",
    "response": "HTTPConnectionPool(host='127.0.0.1', port=8000): Max retries exceeded with url: /datatb/product/add/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fabe00565b0>: Failed to establish a new connection: [Errno 61] Connection refused'))",
    "mutation_type": "extremely_long_value",
    "request_id": 10,
    "timestamp": "2025-04-24T18:38:40.714960",
    "error": "HTTPConnectionPool(host='127.0.0.1', port=8000): Max retries exceeded with url: /datatb/product/add/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fabe00565b0>: Failed to establish a new connection: [Errno 61] Connection refused'))"
  }
]